locale/en/download/package-manager.md

```diff
@@ -356,6 +356,14 @@ Download the [Windows Installer](https://nodejs.org/en/#home-downloadhead) direc
 
 ### Alternatives
 
+Using **[Winget](https://aka.ms/winget-cli)**:
+
+```bash
+winget install OpenJS.NodeJS
+# or for LTS
+winget install OpenJS.NodeJS.LTS
+```
+
```

Comment:
Maybe it would be worthwhile to instruct users to restart their terminal before the `node` CLI command becomes available (or somehow source the user config).

Reply:
Due to this now being shell-agnostic, there wouldn't really be a way to instruct users to restart their terminal or source a user-config in order for the `node` command to become available.

```suggestion

After running one of the two commands above, it may be necessary to restart the terminal emulator before the `node` CLI command would become available.

```